go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1762,7 +1762,7 @@ func flowsForDefaultBridge(bridge *bridgeConfiguration, extraIPs []net.IP) ([]st
 	return dftFlows, nil
 }
 
-func commonFlows(bridge *bridgeConfiguration) ([]string, error) {
+func commonFlows(hostSubnets []*net.IPNet, bridge *bridgeConfiguration) ([]string, error) {
```

Comment:
hostSubnets seems like a misnomer here, this is actually hostCIDRs? A real IP/prefix, not a subnet like 10.244.0.0/24?

Reply:
uhm no no, here we're really looking for the `network address /  prefix`. For instance in a dual stack cluster, I get: `172.18.0.0/16 fc00:f853:ccd:e793::/64`. Is `subnet` correct here?

Reply:
Let's decide what terms to use here for `hostSubnets` above, because I'm not consistent across the code. The options I see are:
- `nodeSubnet`: I have a feeling this is used for the pod subnet
- `nodeCIDR`: not sure
- `hostSubnet`
- `hostCIDR` 

WDYT?

Reply:
if you are getting 172.18.0.0/16, then the term hostSubnet is fine. But across your commit here I think the consistency breaks. Let me walk through it again now that I know this is a subnet.

Reply:
looks like your variable names are consistent now. I think it looks good.